### PR TITLE
Add support for sanity::documentsOf()

### PR DESCRIPTION
**Pull request by j33ty**
Add a new function `sanity::documentsOf()` which lists all the document IDs that are part of a given bundle ID in arg.

### CODE REVIEW

File: src/evaluator/functions.ts [hunk 1]
```diff
@@ -458,6 +458,38 @@ sanity['versionOf'] = async function (args, scope, execute) {
...
 
+// eslint-disable-next-line require-await
+sanity['documentsOf'] = async function (args, scope, execute) {
+  if (!scope.source.isArray()) return NULL_VALUE
+
+  const value = await execute(args[0], scope)
+  if (value.type !== 'string') return NULL_VALUE
+  const baseId = value.data
+
+  // All the document belong to a bundle ID if:
```
Comment (tzhelyazkova) [inline src/evaluator/functions.ts:469]:
```suggestion
  // All the documents belong to a bundle ID if:
```

Comment (tzhelyazkova) [reply src/evaluator/functions.ts:469]: or "A document belongs to a bundle ID if:"

**Review by tzhelyazkova — APPROVED** [review]

File: src/evaluator/functions.ts (hunk 1 above)
Comment (j33ty) [reply src/evaluator/functions.ts]: Fixed in https://github.com/sanity-io/groq-js/pull/254/commits/022c7b2f02a764aca20c74b5a8475107b2723188

**Comment by j33ty** [comment]
@tzhelyazkova another review please.

**Review by tzhelyazkova — APPROVED** [review]

**Comment by tzhelyazkova** [comment]
> @tzhelyazkova another review please.

I approved the last time when I left that comment, so you don't have to depend on that to merge but idk what happened 😅

**Pull request merged**